src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValueNew.cs

```diff
@@ -127,5 +128,10 @@ public static UntypedObjectValue New(IUntypedObject untypedObject)
                 IRContext.NotInSource(new UntypedObjectType()),
                 untypedObject);
         }
+
+        public static ColorValue New(Color value)
```

Comment:
>Color [](http://example.com/codeflow?start=37&length=5)

Does adding this reference to System.Drawing.Color cause a new assembly to get loaded?

(looks like it's defined in System.Drawing.Primitives.dll) 
[https://learn.microsoft.com/en-us/dotnet/api/system.drawing.color?view=net-7.0](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.color?view=net-7.0)

Reply:
As discussed offline, it always gets loaded.

Reply:
Looked at this more ...
we already refer to Color in ColorType. This is just causing it to be loaded sooner (probably because FormulaValue.New is more common public surface)

If this becomes a problem, we can remove System.Drawing.Color and use our own tiny color helper.